modules/QuickEncounter.js

```diff
@@ -1278,6 +1293,9 @@ export class QuickEncounter {
     /* Hook on renderJournalSheet */
     static async onRenderJournalSheet(journalSheet, html) {
         if (!game.user.isGM) {return;}
+		
+		if(journalSheet.cellId) { return; } //ignore GM Screen renders
+		if(journalSheet.enhancedjournal) { return; } //ignore Enhanced Journal renders
```

Comment:
Since Enhanced Journals has a button to activate Quick Encounters I've set it to ignore rendering of any sheet Enhanced Journals is trying to display.
And added the one for GM Screen because it kept coming up when it rendered its own Journal Entries.